dpti/workflows/service/workflow_service_module.py

```
@@ -0,0 +1,137 @@
+from typing import Any, Protocol
+
+from injector import Module, inject, provider
+
+from dpti.workflows.service.file_handler import IOHandler, LocalFileHandler
+from dpti.workflows.service.job_executor import DpdispatcherExecutor, JobExecutor
+from dpti.workflows.service.report_generator import (
+    PrefectReportGenerator,
+    ReportGenerator,
+)
+
+from .file_handler import IOHandler, LocalFileHandler
```

Ruff (0.8.2)

Redefinition of unused `IOHandler` from line 9

(F811)